environments/staging/terraform.yml

```diff
@@ -288,6 +288,48 @@ servers:
     group: "kafka"
     os: bionic
 
+  - server_name: "kafka9-staging"
+    server_instance_type: t3a.medium
+    network_tier: "db-private"
+    az: "a"
+    volume_size: 30
+    volume_encrypted: yes
+    volume_type: "gp3"
+    block_device:
+      volume_size: 1
```

Comment:
1GB is enough?

Reply:
Yes, it is very much less. and due to encryption policy we need a different disk . `10.201.41.86 | CHANGED | rc=0 >>
Filesystem      Size  Used Avail Use% Mounted on
/dev/nvme1n1     79G  163M   75G   1% /opt/data
10.201.42.194 | CHANGED | rc=0 >>
Filesystem      Size  Used Avail Use% Mounted on
/dev/nvme1n1     79G  163M   75G   1% /opt/data
10.201.40.170 | CHANGED | rc=0 >>
Filesystem      Size  Used Avail Use% Mounted on
/dev/nvme1n1     79G  218M   75G   1% /opt/data`